mobile/library/cc/engine_builder.h

```diff
@@ -164,14 +160,14 @@ class EngineBuilder {
   int rtds_timeout_seconds_;
   std::string node_id_;
   absl::optional<NodeLocality> node_locality_ = absl::nullopt;
-  std::string ads_address_ = "";
+  std::string ads_address_ = ""; // make absl:optional?
   int ads_port_;
   std::string ads_jwt_token_;
   int ads_jwt_token_lifetime_seconds_;
   std::string ads_ssl_root_certs_;
-  bool enable_cds_ = false;
   std::string cds_resources_locator_;
   int cds_timeout_seconds_;
+  bool enable_cds_ = false;
```

Comment:
why was this moved 2 lines down?  best to keep it where it was so it doesn't show up in the diffs since we aren't changing anything